scripts/sync-checkly-data.ts

```diff
@@ -63,11 +76,57 @@ const main = async () => {
   if (syncCheckResults) {
     console.log("🔄 Syncing check results...");
 
-    await daemon.syncCheckResults({
+    const checkResults = await daemon.syncCheckResults({
       from: minutesAgo(minutesBack),
       to: new Date(),
     });
 
+    const errorMessages = checkResults.map(getErrorMessageFromCheckResult);
```

Comment:
Let's de-duplicate this with a Set

Reply:
i would skip this for now, makes the code more complicated and doesn't save a lot of money.resources in the long run.